paasta_tools/chronos_serviceinit.py

```diff
@@ -330,22 +320,26 @@ def status_chronos_jobs(client, jobs, job_config, verbose):
         desired_state = job_config.get_desired_state_human()
         output.append("Desired:    %s" % desired_state)
         for job in jobs:
-            running_tasks = get_running_tasks_from_frameworks(job["name"])
-            output.append(format_chronos_job_status(client, job, running_tasks, verbose))
+            running_task_count = len(select_tasks_by_id(get_cached_list_of_running_tasks_from_frameworks(),
```

Comment:
Maybe something like this works:
```
time_cache(ttl=600)(get_running_tasks_from_frameworks(job["name"]))
```
or it might be?
```
time_cache(get_running_tasks_from_frameworks(job["name"]))(ttl=600)
```
I have a memory that something like that is possible, and it would avoid duplicating the functions just to turn on the caching?